packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -962,3 +962,44 @@ export const MQA_STATS = {
     systemMaximumCPU: 0
   }
 };
+
+export const VIDEO_RESOLUTIONS = {
```

Comment:
Should we have a config for what max resolution we should receive

Reply:
All of these configs have max resolutions set, so I'm not sure what you are asking.

Reply:
Was talking about default config . If someone wants to start a meeting by default with lower resolution or video

Reply:
Good idea! I've created another task for changing settings pre-join as this one is scope towards during call changes. https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-133817